spec/models/pet_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+describe Pet, type: :model do
+    describe "validations" do
+        it {should validate_presence_of :name}
+        it {should validate_presence_of :image}
+        it {should validate_presence_of :description}
+        it {should validate_presence_of :age}
+        it {should validate_presence_of :sex}
+        it {should validate_presence_of :adoption_status}
+    end
+
+    describe "relationships" do
+        it {should belong_to :shelter}
+    end
```

Comment:
Model test added to make sure that the pet class has the appropriate relationships and validations defined.